src/features/lightspeed/api.ts

```diff
@@ -114,7 +128,6 @@ export class LightSpeedAPI {
     } catch (error) {
       const err = error as AxiosError;
       vscode.window.showErrorMessage(retrieveError(err));
-      this._completionRequestInProgress = false;
```

Comment:
Why is this line removed? The return in next line won't run the finally block.

Reply:
Pls look at my response above. Thanks!